app/config/FrontendAppConfig.scala

```diff
@@ -107,7 +107,11 @@ class FrontendAppConfig @Inject()(configuration: Configuration, servicesConfig:
   lazy val aftOverviewUrl: String = s"$aftUrl${configuration.get[String](path = "urls.aftOverview")}"
   lazy val isAftNonZero: String = s"$aftUrl${configuration.get[String](path = "urls.isAftNonZero")}"
   lazy val psaFinancialStatementUrl: String = s"$aftUrl${configuration.get[String](path = "urls.psaFinancialStatement")}"
-  lazy val podsNewFinancialCredits: Boolean = configuration.getOptional[Boolean]("pods.new.financial.credits").getOrElse(false)
+  lazy val podsNewFinancialCredits: Boolean = {
+
+    val bool = configuration.getOptional[Boolean]("pods.new.financial.credits").getOrElse(false)
```

Comment:
we can remove **val bool** and return directly **configuration.getOptional[Boolean]("pods.new.financial.credits").getOrElse(false)**

Reply:
Removed